File: Domain/Product.cs
using System;

namespace Domain
{
    public class Product
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public int Cost { get; set; }
        public string Photo { get; set; }
        public string Address { get; set; }
        public long CardNumber { get; set; }
    }
}

File: client-application/src/app/models/product.ts
export interface Product {
    id: string;
    title: string;
    cost: number;
    photo: string;
    address: string;
    cardNumber: string;
}

export interface ProductForPost {
    id: string;
    title: string;
    cost: number;
    photo: string;
    address: string;
    cardNumber: bigint;
}

export interface ProductInBox {
    id: string;
    title: string;
    cost: number;
    photo: string;
    count: number;
    costAll: number;
}

export class ProductinBox {
    id: string = '';
    title: string = '';
    cost: number = 0;
    photo: string = '';
}

export interface ManagerLogs {
    id: string;
    title: string;
    numberOfGood: number;
    cost: number;
    photo: string;
    count: number;
    address: string;
    cardNumber: number;
    payed: string;
    statusManager: string;
}